qemu/tests/rng_hotplug.py

```diff
@@ -28,9 +28,10 @@ def run(test, params, env):
 
     def get_rng_id(vm):
         device_list = []
+        rng_driver = params.get("rng_driver")
```

Comment:
Should set default driver for other platforms, otherwise, they will fail.

Reply:
RHEL 8.6:

 (01/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads.s390-virtio: STARTED
 (01/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads.s390-virtio: PASS (741.89 s)
 (02/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rh_kernel_update.s390-virtio: STARTED
 (02/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rh_kernel_update.s390-virtio: PASS (77.35 s)
 (03/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads.s390-virtio: STARTED
 (03/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads.s390-virtio: PASS (740.22 s)
 (04/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rh_kernel_update.s390-virtio: STARTED
 (05/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_bat.s390-virtio: STARTED
 (05/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_bat.s390-virtio: PASS (27.10 s)
 (06/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_bat.s390-virtio: STARTED
 (06/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_bat.s390-virtio: PASS (24.11 s)
 (07/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_egd.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_bat.s390-virtio: STARTED
 (07/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_egd.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_bat.s390-virtio: PASS (31.35 s)
 (08/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_read_longtime.s390-virtio: STARTED
 (08/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_read_longtime.s390-virtio: PASS (3623.18 s)
 (13/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_maxbytes_period.negative_test.maxbytes_negative.s390-virtio: STARTED
 (13/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_maxbytes_period.negative_test.maxbytes_negative.s390-virtio: PASS (24.49 s)
 (14/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_maxbytes_period.negative_test.period_0.s390-virtio: STARTED
 (14/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_maxbytes_period.negative_test.period_0.s390-virtio: PASS (24.03 s)
 (15/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_maxbytes_period.negative_test.period_negative.s390-virtio: STARTED
 (15/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_maxbytes_period.negative_test.period_negative.s390-virtio: PASS (24.63 s)
 (16/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_stress.two_device.s390-virtio: STARTED
 (16/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_stress.two_device.s390-virtio: PASS (27.29 s)
 (17/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_stress.one_device.s390-virtio: STARTED
 (17/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_stress.one_device.s390-virtio: PASS (25.04 s)
 (18/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_stop_continue.before_bg_test.s390-virtio: STARTED
 (18/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_stop_continue.before_bg_test.s390-virtio: PASS (29.53 s)
 (19/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_stop_continue.during_bg_test.s390-virtio: STARTED
 (19/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_stop_continue.during_bg_test.s390-virtio: PASS (73.07 s)
 (20/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_stop_continue.after_bg_test.s390-virtio: STARTED
 (20/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_stop_continue.after_bg_test.s390-virtio: PASS (31.79 s)
 (21/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_shutdown.before_bg_test.s390-virtio: STARTED
 (21/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_shutdown.before_bg_test.s390-virtio: PASS (46.21 s)
 (22/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_shutdown.during_bg_test.s390-virtio: STARTED
 (22/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_shutdown.during_bg_test.s390-virtio: PASS (23.72 s)
 (23/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_shutdown.after_bg_test.s390-virtio: STARTED
 (23/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_shutdown.after_bg_test.s390-virtio: PASS (24.10 s)
 (24/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_reboot.before_bg_test.s390-virtio: STARTED
 (24/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_reboot.before_bg_test.s390-virtio: PASS (35.88 s)
 (25/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_reboot.during_bg_test.s390-virtio: STARTED
 (25/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_reboot.during_bg_test.s390-virtio: PASS (65.45 s)
 (26/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_reboot.after_bg_test.s390-virtio: STARTED
 (26/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_reboot.after_bg_test.s390-virtio: PASS (38.00 s)
 (27/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_system_reset.before_bg_test.s390-virtio: STARTED
 (27/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_system_reset.before_bg_test.s390-virtio: PASS (55.83 s)
 (28/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_system_reset.during_bg_test.s390-virtio: STARTED
 (28/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_system_reset.during_bg_test.s390-virtio: PASS (83.18 s)
 (29/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_system_reset.after_bg_test.s390-virtio: STARTED
 (29/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_system_reset.after_bg_test.s390-virtio: PASS (58.28 s)
 (30/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_live_migration.before_bg_test.s390-virtio: STARTED
 (30/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_live_migration.before_bg_test.s390-virtio: PASS (49.30 s)
 (31/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_live_migration.during_bg_test.s390-virtio: STARTED
 (31/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_live_migration.during_bg_test.s390-virtio: PASS (78.69 s)
 (32/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_live_migration.after_bg_test.s390-virtio: STARTED
 (32/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_live_migration.after_bg_test.s390-virtio: PASS (48.75 s)
 (33/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_host_guest_read.s390-virtio: STARTED
 (33/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_host_guest_read.s390-virtio: PASS (24.36 s)
 (34/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_hotplug.multi_rngs.s390-virtio: STARTED
 (34/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_hotplug.multi_rngs.s390-virtio: PASS (116.31 s)
 (35/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_hotplug.one_rng.s390-virtio: STARTED
 (35/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_hotplug.one_rng.s390-virtio: PASS (67.59 s)
 (36/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_hotplug.one_rng.with_reboot.s390-virtio: STARTED
 (36/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_hotplug.one_rng.with_reboot.s390-virtio: PASS (96.91 s)
 (37/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_hotplug.one_rng.with_system_reset.s390-virtio: STARTED
 (37/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_hotplug.one_rng.with_system_reset.s390-virtio: PASS (115.40 s)
 (38/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_hotplug.one_rng.with_shutdown_after_unplug.s390-virtio: STARTED
 (38/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_hotplug.one_rng.with_shutdown_after_unplug.s390-virtio: PASS (68.57 s)
 (39/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_hotplug.one_rng.with_shutdown_after_plug.s390-virtio: STARTED
 (39/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_hotplug.one_rng.with_shutdown_after_plug.s390-virtio: PASS (46.16 s)
 (40/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_hotplug.repeat_in_loop.s390-virtio: STARTED
 (40/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_builtin.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_hotplug.repeat_in_loop.s390-virtio: PASS (2314.08 s)
 (41/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_read_longtime.s390-virtio: STARTED
 (41/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_read_longtime.s390-virtio: PASS (3623.20 s)
 (46/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_maxbytes_period.negative_test.maxbytes_negative.s390-virtio: STARTED
 (46/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_maxbytes_period.negative_test.maxbytes_negative.s390-virtio: PASS (24.60 s)
 (47/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_maxbytes_period.negative_test.period_0.s390-virtio: STARTED
 (47/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_maxbytes_period.negative_test.period_0.s390-virtio: PASS (25.05 s)
 (48/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_maxbytes_period.negative_test.period_negative.s390-virtio: STARTED
 (48/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_maxbytes_period.negative_test.period_negative.s390-virtio: PASS (24.28 s)
 (49/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_stress.two_device.s390-virtio: STARTED
 (49/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_stress.two_device.s390-virtio: PASS (28.83 s)
 (50/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_stress.one_device.s390-virtio: STARTED
 (50/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_stress.one_device.s390-virtio: PASS (25.18 s)
 (51/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_stop_continue.before_bg_test.s390-virtio: STARTED
 (51/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_stop_continue.before_bg_test.s390-virtio: PASS (30.46 s)
 (52/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_stop_continue.during_bg_test.s390-virtio: STARTED
 (52/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_stop_continue.during_bg_test.s390-virtio: PASS (77.58 s)
 (53/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_stop_continue.after_bg_test.s390-virtio: STARTED
 (53/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_stop_continue.after_bg_test.s390-virtio: PASS (33.25 s)
 (54/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_shutdown.before_bg_test.s390-virtio: STARTED
 (54/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_shutdown.before_bg_test.s390-virtio: PASS (46.66 s)
 (55/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_shutdown.during_bg_test.s390-virtio: STARTED
 (55/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_shutdown.during_bg_test.s390-virtio: PASS (29.85 s)
 (56/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_shutdown.after_bg_test.s390-virtio: STARTED
 (56/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_shutdown.after_bg_test.s390-virtio: PASS (23.61 s)
 (57/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_reboot.before_bg_test.s390-virtio: STARTED
 (57/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_reboot.before_bg_test.s390-virtio: PASS (36.88 s)
 (58/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_reboot.during_bg_test.s390-virtio: STARTED
 (58/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_reboot.during_bg_test.s390-virtio: PASS (69.55 s)
 (59/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_reboot.after_bg_test.s390-virtio: STARTED
 (59/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_reboot.after_bg_test.s390-virtio: PASS (38.59 s)
 (60/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_system_reset.before_bg_test.s390-virtio: STARTED
 (60/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_system_reset.before_bg_test.s390-virtio: PASS (56.05 s)
 (61/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_system_reset.during_bg_test.s390-virtio: STARTED
 (61/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_system_reset.during_bg_test.s390-virtio: PASS (82.27 s)
 (62/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_system_reset.after_bg_test.s390-virtio: STARTED
 (62/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_system_reset.after_bg_test.s390-virtio: PASS (57.00 s)
 (63/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_live_migration.before_bg_test.s390-virtio: STARTED
 (63/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_live_migration.before_bg_test.s390-virtio: PASS (48.46 s)
 (64/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_live_migration.during_bg_test.s390-virtio: STARTED
 (64/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_live_migration.during_bg_test.s390-virtio: PASS (78.01 s)
 (65/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_live_migration.after_bg_test.s390-virtio: STARTED
 (65/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.viorng_in_use.with_live_migration.after_bg_test.s390-virtio: PASS (48.02 s)
 (66/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_host_guest_read.s390-virtio: STARTED
 (66/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_host_guest_read.s390-virtio: PASS (24.60 s)
 (67/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_hotplug.multi_rngs.s390-virtio: STARTED
 (67/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_hotplug.multi_rngs.s390-virtio: PASS (116.23 s)
 (68/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_hotplug.one_rng.s390-virtio: STARTED
 (68/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_hotplug.one_rng.s390-virtio: PASS (68.42 s)
 (69/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_hotplug.one_rng.with_reboot.s390-virtio: STARTED
 (69/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_hotplug.one_rng.with_reboot.s390-virtio: PASS (97.10 s)
 (70/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_hotplug.one_rng.with_system_reset.s390-virtio: STARTED
 (70/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_hotplug.one_rng.with_system_reset.s390-virtio: PASS (115.20 s)
 (71/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_hotplug.one_rng.with_shutdown_after_unplug.s390-virtio: STARTED
 (71/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_hotplug.one_rng.with_shutdown_after_unplug.s390-virtio: PASS (68.20 s)
 (72/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_hotplug.one_rng.with_shutdown_after_plug.s390-virtio: STARTED
 (72/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_hotplug.one_rng.with_shutdown_after_plug.s390-virtio: PASS (48.16 s)
 (73/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_hotplug.repeat_in_loop.s390-virtio: STARTED
 (73/73) Host_RHEL.m9.u0.nographic.virtio_rng.rng_random.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.rng_hotplug.repeat_in_loop.s390-virtio: PASS (2335.11 s)